connection.go

```diff
@@ -26,6 +26,12 @@ const (
 	statementTypeIDDelete           = statementTypeIDDml + int64(0x300)
 	statementTypeIDMerge            = statementTypeIDDml + int64(0x400)
 	statementTypeIDMultiTableInsert = statementTypeIDDml + int64(0x500)
+
+	statementTypeIDDdl     = int64(0x6000)
+	statementTypeIDCreate  = statementTypeIDDdl + int64(0x101)
+	statementTypeIDComment = statementTypeIDDdl + int64(0x200)
+	statementTypeIDDrop    = statementTypeIDDdl + int64(0x300)
+	statementTypeIDAlter   = statementTypeIDDdl + int64(0x401)
```

Comment:
why you need these?

Reply:
It was requested in an open issue.

Reply:
It would be the best if you add this when it is required.